dftimewolf/metawolf/metawolf.py

```diff
@@ -962,7 +967,7 @@ def UpdateSessionSettable(
 
     prev_value = settable.GetValue()
     value = utils.CastToType(updated_value, settable.type)
-    if not value:
+    if value is None:
```

Comment:
What if we're trying to clear a parameter and set it to `None`?

Reply:
Also fine
```
$b552a4$aws_forensics> reload
remote_profile_name: dasd --> None
remote_zone: asdasd --> None
incident_id: sdads --> None
all_volumes: True --> False
boot_volume_size: 0 --> 50
```